WindDirectionControl/WindDirectionControl.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Agate styled WindDirectionControl component and behavior.
+ *
+ * @example
+ * <WindDirectionControl />
+ *
+ * @module agate/WindDirectionControl
+ * @exports WindDirectionControl
+ * @exports WindDirectionControlBase
+ * @exports WindDirectionControlDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import ArcPicker from '../ArcPicker';
+import Icon from '../Icon';
+import Skinnable from '../Skinnable';
+
+import css from './WindDirectionControl.module.less';
+
+/**
+ * An Agate component for displaying Wind Direction Control {@link agate/WindDirectionControl}.
+ *
+ * @class WindDirectionControl
+ * @memberof agate/WindDirectionControl
+ * @ui
+ * @public
+ */
+const WindDirectionControlBase = kind({
+	name: 'WindDirectionControlBase',
+
+	propTypes: /** @lends agate/WindDirectionControl.WindDirectionControlBase.prototype */ {
+		/**
+		 * Called when value is changed.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		onChange: PropTypes.func,
+
+		/**
+		 * State of WindDirectionControl.
+		 *
+		 * @type {('airDown'|'airRight'|'airUp')}
+		 * @default 'airDown'
+		 * @public
+		 */
+		value:  PropTypes.oneOf(['airDown', 'airRight', 'airUp'])
+	},
+
+	defaultProps: {
+		value: 'airDown'
+	},
+
+	styles: {
+		css,
+		className: 'windDirectionControl'
+	},
+
+	computed: {
+		componentIcon: (props) => {
+			const {value} = props;
+
+			switch (value) {
+				case 'airDown':
+					return 'airdown';
+				case 'airRight':
+					return 'airup';
+				case 'airUp':
+					return 'airright';
+			}
```

Comment:
How about `return value.toLowerCase();`?
In adddition, we might return 'airup' for 'airUp' and 'airright' for 'airRight', right?

Reply:
Well, this can't be changed. When value = 'airRight' I need to display this icon: 
![airup](https://user-images.githubusercontent.com/63335823/95321495-c7389e00-08a3-11eb-8cdf-8640469efc0a.PNG)
which is named 'airup'. Maybe if the naming for this icon will be changed, we can go with our approach. I hope I made myself clear 😄

Reply:
Now I understand what the problem is. I fixed the icon map and pushed the commit. Please let me know if something wrong.

Reply:
Everything looks great.